integration/helpers/build.sh

```diff
@@ -2,7 +2,7 @@
 
 build_changed() {
     hab pkg install core/ruby
-    PATH=$PATH:$(hab pkg path core/ruby)/bin
+    PATH=$(hab pkg path core/ruby)/bin:$PATH
```

Comment:
I don't know if this could cause the same problem, but we also construct PATH this way around in this script: https://github.com/chef/automate/blob/3757e9fdfec19d435860d46506e6f51611a3bea4/scripts/verify_build.sh#L18